_posts/2023-02-13-announcing-duckdb-070.md

```diff
@@ -139,10 +139,10 @@ SELECT * FROM t1 POSITIONAL JOIN t2;
 
 **Query Building.** This release introduces easier incremental query building using the Python API by allowing relations to be queried. This allows you to decompose long SQL queries into multiple smaller SQL queries, and allows you to easily inspect query intermediates.
 
-```
-import duckdb
-lineitem = duckdb.sql('FROM lineitem.parquet')
-lineitem.limit(3).show()
+```py
```

Comment:
I removed the `py` intentionally from the code here because the tables end up in all-red formatting:

![Screenshot 2023-02-13 at 12 36 44](https://user-images.githubusercontent.com/3978469/218447828-ad10eeff-fdc6-4c39-bfa7-0ad60f3733d0.png)

Perhaps I just have an out-dated formatter or so?